src/main/java/de/tum/in/www1/artemis/domain/LearningGoal.java

```diff
@@ -74,6 +79,14 @@ public void setDescription(String description) {
         this.description = description;
     }
 
+    public Integer getMasteryThreshold() {
```

Comment:
This can be an int since null can no longer get returned
```suggestion
    public int getMasteryThreshold() {
```